tests/runner.sh

```diff
@@ -1,5 +1,9 @@
 source ../scripts/init
 
-./test_init_var.sh
-./test_bash_utils.sh
-./test_devenv_impl.sh
+final_ret=0
+
+./test_init_var.sh ; if [ $? != 0 ] ; then final_ret=1 ; fi
```

Comment:
Need to keep track of $? from the sub-scripts and exit with that errno.  Otherwise failures in the sub-scripts do not flag a failure in Github action!